alot/commands/envelope.py

```diff
@@ -324,10 +324,9 @@ def openEnvelopeFromTmpfile():
 
             # get input
             # tempfile will be removed on buffer cleanup
-            f = open(self.envelope.tmpfile.name)
-            enc = settings.get('editor_writes_encoding')
-            template = string_decode(f.read(), enc)
-            f.close()
+            with open(self.envelope.tmpfile.name) as f:
+                enc = settings.get('editor_writes_encoding')
```

Comment:
I know it was in the middle before but does this line need to be in the middle or could it also be above the with block?